packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -331,11 +331,9 @@ export function computeExpirationForFiber(
 
   let expirationTime;
   if (suspenseConfig !== null) {
-    // Compute an expiration time based on the Suspense timeout.
-    expirationTime = computeSuspenseExpiration(
-      currentTime,
-      suspenseConfig.timeoutMs | 0 || LOW_PRIORITY_EXPIRATION,
-    );
+    // If there's a SuspenseConfig, treat this as a concurrent update regardless
+    // of the priority.
+    expirationTime = computeAsyncExpiration(currentTime);
```

Comment:
Split into two "lanes" depending on how large the timeout is.

- Smallish numbers get currentTime + 10sec
- "Infinite" numbers get really big number